core/DataAccess/Model.php

```diff
@@ -203,7 +203,13 @@ public function updateArchiveAsInvalidated($archiveTable, $idSites, $allPeriodsT
 
         $dummyArchives = [];
         foreach ($idSites as $idSite) {
-            $siteCreationTime = Date::factory(Site::getCreationDateFor($idSite));
+            try {
+                $siteCreationTime = Site::getCreationDateFor($idSite);
+            } catch (\Exception $ex) {
+                continue;
+            }
+
+            $siteCreationTime = Date::factory($siteCreationTime);
```

Comment:
fyi @diosmosis  in this case $siteCreationTime might be not defined or pointing to the time of a previous site?

Reply:
actually forget it I see you continue above.

Reply:
It should skip to the next iteration if it can't be set?

Reply:
Totally. sorry about that

Reply:
oh, github showed me the first comment, but not the second...